Makefile

```diff
@@ -41,6 +41,10 @@ ENVTEST_K8S_VERSION = 1.24
 
 INTEGRATION_TARGET ?= ./test/integration/...
 
+E2E_TARGET ?= ./test/e2e/...
+
+E2E_KIND_VERSION ?= kindest/node:v1.24.6
```

Comment:
@kerthcet I saw you mentioned we should test against 1.22 but I can't find it on the review.  I took 1.24.6 because we were using 1.24 for kubebuilder assets.

Reply:
maybe we can do 1.23 https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md#before-you-begin

Reply:
Sure.  This can be a followup but do we know how we will target e2e tests with different Kubernetes versions?

I see that the presubmits has various releases but I'm not sure when we target new versions of Kubernetes and Kueue.

Reply:
We don't target a specific version because we don't have E2E tests yet :) Using a newer client is not the same as targeting a version.

I think we should start in 1.23 because it has all the features that we are using. And maybe we should always target max(min_version_with_features_used, min_upstream_supported_version).

The later is currently 1.23 as well https://kubernetes.io/releases/version-skew-policy/#supported-versions

Reply:
sgtm.

Reply:
Updated.